### PR TITLE
main/zmap: build with json and redis support

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

File: main/zmap/APKBUILD [hunk 1]
```diff
@@ -3,14 +3,14 @@
 # Maintainer: 
 pkgname=zmap
 pkgver=2.1.1
-pkgrel=3
+pkgrel=4
 pkgdesc="A high performance internet scanner"
 url="https://www.zmap.io/"
 arch="all"
 license="Apache-2.0"
 depends=""
```
Comment (maxice8) [inline main/zmap/APKBUILD]: empty variables can be removed for brevity

**Comment by J0WI** [comment]
Still waiting for merge ⏳

**Comment by ncopa** [comment]
Build fails:
```
/usr/bin/abuild: line 36: ./test_big_group.sh: not found
>>> ERROR: zmap: check failed
```

**Comment by J0WI** [comment]
ac82d11fe951fd90c3cda76c17fece233a7d9569

**Pull request closed without merging**